app/models/task.rb

```diff
@@ -10,4 +12,16 @@ def self.get_all_completed_tasks
   def self.get_all_incomplete_tasks
     Task.where(completed: false).to_json
   end
+
+  def self.create_task(params) 
```

Comment:
This looks good. This  `if model.save` pattern is super common in rails controllers